project/src/main/scala/org/broadinstitute/clio/integrationtest/ClioIntegrationTestRunner.scala

```diff
@@ -132,6 +163,25 @@ class ClioIntegrationTestRunner(testClassesDirectory: File,
     val processLogger = ProcessLogger(logFunction)
     process ! processLogger
   }
+
+  /**
+    * Run a command asynchronously using scala.sys.process.
+    *
+    * @param command     The command to run.
+    * @param cwd         Directory to run from.
+    * @param extraEnv    Other environment variables to pass to the command.
+    * @param logFunction A function implementing logging.
+    */
+  private def runCommandAsync(command: Seq[String],
+                              cwd: File,
+                              extraEnv: Seq[(String, String)],
+                              logFunction: String => Unit): Unit = {
+    import scala.sys.process._
+    val process = Process(command, cwd, extraEnv: _*)
+    val processLogger = ProcessLogger(logFunction)
+    process run processLogger
+    ()
+  }
```

Comment:
If we do keep separate docker-compose calls, I think it'd be better to have this function return a `Future` instead of `Unit`. That way we can detect when the command fails, vs this which is more fire-and-forget.